photon.json

```diff
@@ -63,6 +63,9 @@
     },
     {
       "type": "shell",
+      "environment_vars": [
+        "VEBA_VERSION={{ user `VEBA_VERSION` }}"
```

Comment:
Where does `{{ user `VEBA_VERSION` }}` come from? Why do we need this variable and cannot reuse the one above/similar approach?

Reply:
Good question and yes it may not look apparent :)

Let me explain. 

Since we're building the appliance from scratch, we don't have `/etc/veba-release` file at all, since there's no OS, so its sort of a chicken/egg problem and hence we can't use the method above which executes AFTER customer deploys the appliance and has the file.

In this phase, we need to grab the version from `VERSION` file which is passed in from https://github.com/vmware-samples/vcenter-event-broker-appliance/blob/development/build.sh#L14 and then this gets exposed as an ENV variable within the section of the pre-installer within Packer. So this translate to the $VEBA_VERSION variable containing the value to then determine the VMware Event Router tag to pull (for air-gap). This all happens as part of installing the OS and setting things up.

Reply:
Gotcha, if it’s coming from `VERSION` we’re good.